draft-ietf-quic-transport.md

```diff
@@ -5279,14 +5279,14 @@ ACK Delay:
 
 : A variable-length integer representing the time delta in microseconds between
   when the ACK frame was sent and when the largest acknowledged packet, as
-  indicated in the Largest Acknowledged field, was received by this peer.  The
-  value of the ACK Delay field is scaled by multiplying the encoded value by 2
-  to the power of the value of the ack_delay_exponent transport parameter set by
-  the sender of the ACK frame; see {{transport-parameter-definitions}}.  Scaling
-  in this fashion allows for a larger range of values with a shorter encoding at
-  the cost of lower resolution.  Because the receiver doesn't use the value in
-  the ACK Delay field for Initial and Handshake packets, a sender SHOULD send a
-  value of 0.
+  indicated in the Largest Acknowledged field, was received by this peer.  This
+  acknowledgement delay in the ACK Delay field is scaled by multiplying the
+  encoded value by 2 to the power of the value of the ack_delay_exponent
+  transport parameter set by the sender of the ACK frame; see
+  {{transport-parameter-definitions}}.  Scaling in this fashion allows for a
+  larger range of values with a shorter encoding at the cost of lower
+  resolution.  Because the receiver doesn't use the value in the ACK Delay field
+  of Initial and Handshake packets, a sender SHOULD send a value of 0.
```

Comment:
This text was already confusing, and probably needs more rework. For this PR though, I would suggest something like: "The ACK Delay field carries an encoded acknowledgement delay. The delay is decoded by multiplying the value in the ACK Delay field by 2 ..."